src/Common/tests/System/Net/RemoteServerQuery.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Test.Common
+{
+    internal class RemoteServerException : Exception
+    {
+        private const string Description = "Likely external issue with remote server";
+
+        public RemoteServerException()
+            : this(null, null) { }
+
+        public RemoteServerException(string server)
```

Comment:
Can we remove this overload? - to force everyone to use the original exception as well?

Reply:
sure